### PR TITLE
docs: update README to be working with latest raspberry light OS

### PR DESCRIPTION
**Pull request by mohoff**
Update README based on latest TextClock installation steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,55 @@
...
+```
+
+Some of these tools might be pre-installed
+
+### Uninstall screensaver
+
+Once the clock was started, the screensaver kicks in after a few minutes. To prevent this, the screensaver can be uninstalled.
+
+```sh
+sudo apt remove xscreensaver
```
Comment (thetillhoff) [inline README.md]: If you add `-y` no confirmation is required.

File: README.md [hunk 2]
```diff
@@ -33,28 +61,50 @@ StartupNotify=false
...
+```sh
+sudo dpkg-reconfigure tzdata
+// or
+sudo raspi-config
+```
+
 ### Enable automatic time update
-`sudo timedatectl set-ntp true`
+
+Make sure the ntp daemon and `systemd-timesyncd` are running by checking `service status <SERVICE>`.
```
Comment (thetillhoff) [inline README.md]: `ntp`? Or how is the service named?

**Review by thetillhoff — APPROVED** [review]
LGTM apart from the two comments I made. Great job 👌

File: README.md (hunk 2 above)
Comment (mohoff) [reply README.md]: `ntp` I think. Put in backticks

**Pull request merged**